client/js/navigation/BottomNavigationLayout.js

```diff
@@ -1 +1,34 @@
 // bottom navigation
+import React from 'react';
+import {
+  createStackNavigator,
+  createBottomTabNavigator
+} from 'react-navigation';
+import BeerScreen from '../screens/Beer';
+import CardScreen from '../screens/Card';
+import EventsScreen from '../screens/Events';
+import HomeScreen from '../screens/Home';
+import StoresScreen from '../screens/Stores';
```

Comment:
are you going to create stacks for the rest of your screens?

Reply:
Yes, but i want to do it in my branch called "jy-feature-navigation" instead of this branch, "jy-feature-create-folders"

Reply:
in that case, you should have just created folders and empty files for this pull request.